koji/vars/external_repos.yml

```diff
@@ -0,0 +1,128 @@
+---
+external_repos:
+  - name: puppetlabs-pc1-fedora-24
+    url: file:///mnt/koji/external-repos/www/puppetlabsf24-$arch/RPMS.pc1/
+  - name: rhel-7.0-server
+    url: file:///mnt/koji/external-repos/rhel7-$arch/7.0/
+  - name: epel-7
+    url: file:///mnt/koji/external-repos/www/epel7-$arch/RPMS.epel/
+  - name: fedora-23
+    url: file:///mnt/koji/external-repos/www/fedora23-$arch/RPMS.os/
+  - name: fedora-23-updates
+    url: file:///mnt/koji/external-repos/www/fedora23-$arch/RPMS.updates/
```

Comment:
Do we still have F23? I thought we cleaned that up a while back. Same with the other older Fedora versions.

Reply:
Nope, but see description :)

Reply:
I just assumed we actually removed the repos from disk.

Reply:
pretty sure we did, but given how manual its all been, the external repos in Koji were not cleaned up. I wonder what we should do about tags that these external repos are attached to as well (when we remove them).